projects/plugins/boost/wp-js-data-sync.php

```diff
@@ -136,3 +139,11 @@ function jetpack_boost_initialize_datasync() {
 
 $entry = new Modules_State_Entry();
 jetpack_boost_register_option( 'modules_state', $modules_state_schema, $entry );
+
+/**
+ * Register Minify Excludes stores.
+ */
+$js_excludes_entry  = new Minify_Excludes_State_Entry( 'minify_js_excludes' );
+$css_excludes_entry = new Minify_Excludes_State_Entry( 'minify_css_excludes' );
+jetpack_boost_register_option( 'minify_js_excludes', Schema::as_string()->fallback( implode( ',', Minify_JS::$default_excludes ) ), $js_excludes_entry );
+jetpack_boost_register_option( 'minify_css_excludes', Schema::as_string()->fallback( implode( ',', Minify_CSS::$default_excludes ) ), $css_excludes_entry );
```

Comment:
If we inline the fallback here to be a plain string, that's going to be the fallback and you can use `jetpack_boost_ds_get()` anywhere even when the values aren't present in the database, so extra value checking in functions like `jetpack_boost_page_optimize_js_exclude_list` is unnecessary.

Reply:
Updated in a13eaf806dd706efa170bc8b8da768e7586dfb44.